rebar.config

```diff
@@ -1,14 +1,14 @@
 {erl_opts, [debug_info]}.
 
 {deps, [ {aebytecode, {git, "https://github.com/aeternity/aebytecode.git",
-                            {branch,"fortuna"}}}
+                             {tag,"v2.0.1"}}}
```

Comment:
This reminds me again, we need to think about versioning (and documentation, and release notes?!)...